src/util/makeCancelable.spec.ts

```diff
@@ -0,0 +1,42 @@
+import makeCancelable from "./makeCancelable";
+
+describe("makeCancelable", () => {
+  it("should resolve when promise resolves", async () => {
+    const future = new Promise(resolve => {
+      setTimeout(() => resolve("well done!"), 1000);
```

Comment:
To have better tests (don't wait 1sc for each test) you can use this -> https://jestjs.io/docs/en/timer-mocks